src/server/game/Miscellaneous/Formulas.h

```diff
@@ -228,6 +229,16 @@ namespace Trinity
             sScriptMgr->OnGroupRateCalculation(rate, count, isRaid);
             return rate;
         }
+
+        inline float xp_on_weekend_rate(uint32 initial_rate = 1.0f)
+        {
+            if(IsEventActive(sWorld->getIntConfig(CONFIG_RATE_XP_WEEKEND_EVID)))
```

Comment:
Just noticed I gave the uint32 initial_rate a float value, should be a float rate...